troposphere/themes/troposphere_theme/scss/_nav.scss

```diff
@@ -23,8 +23,13 @@
         background-color: darken($navbar-default-bg, 10%);
     }
     .navbar-brand {
-        background: url(../images/mini_logo.png) no-repeat 10px center;
-        background-size: 30px 30px;
+        position: relative;
+        width: 200px;
+        height: 50px;
+        background: url(../images/mini_logo.png) no-repeat;
```

Comment:
If I understand the question, Cyverse logo includes text so it is much longer. I made it a little taller because it looks better to me. (see screenshot for visual)